plugins/cocoapods/README.md

```diff
@@ -0,0 +1,55 @@
+# Cocoapods Plugin
+
+Use auto to version your [Cocoapod](https://cocoapods.org/), and push to your specs repository!
+
+## Installation
+
+This plugin is not included with the `auto` CLI installed via NPM. To install:
+
+```sh
+npm i --save-dev @auto-it/cocoapods
+# or
+yarn add -D @auto-it/cocoapods
+```
+
+## Usage
+
+```jsonc
+{
+  "plugins": [
+    [
+      "cocoapods",
+      {
+        // Required, the relative path to your podspec file
+        "podspecPath": "./Test.podspec",
+        // Optional, the specs repo to push to
+        "specsRepo": "https://github.com/intuit/TestSpecs.git"
+      }
+    ]
+    // other plugins
+  ],
+  // THIS IS REQUIRED
+  // Cocoapods cannot work with tags in the format v1.0.0
+  "noVersionPrefix": true
+}
+```
+
+## Requirements
+
+### General
+
+- The machine running this plugin must have the [Cocoapods](https://cocoapods.org/) `pod` CLI installed already.
+
+- Your `podspec` file must pass `pod lib lint` in order for publishing to a Specs repository to work.
+
+### Versioning
+
+[Cocoapods](https://cocoapods.org/) does not work with the version prefix of `v1.0.0` so all versions must be the plain semver number (`1.0.0`). You are required to set the `noVersionPrefix` setting in your auto configuration.
```

Comment:
Actually I think this might be the perfect use case for the [`modifyConfig` hook](https://intuit.github.io/auto/pages/writing-plugins.html#modifyconfig). This way a user never has to worry about it!

Reply:
I guess i should have read all the docs, I can implement this